src/Layout/Callout.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import styled from 'styled-components';
+import { VARIANT } from '../Theme';
+
+export interface CalloutProps {
+  /**
+ * Allows the user to set the background color by passing in a variant property from the VARIANT enum
+ * @default VARIANT.INFO
+ */
+  variant?: VARIANT;
+  /**
+   * The aria role of the displayed callout. A list of the different role
+   * values that should be used can be found in the w3 docs:
+   * https://www.w3.org/TR/wai-aria-1.1/#live_region_roles
+   * The default value role is 'note'.
+   */
+  role?: string;
```

Comment:
There's [an `AriaRole` type][1] defined in the `@types/react` package, which we could use here to ensure that the passed value is actually a valid role.

```ts
import React, { AriaRole } from 'react';

export interface CalloutProps {
  role?: AriaRole;
}
```

[1]: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/298040491bdf086e6fbfe3c9eb12fbe96c90b7fc/types/react/index.d.ts#L2815-L2885